coding-projects-ai-guidelines.md

```
@@ -142,196 +116,169 @@ Which approach would you prefer, or would you like me to explore other options?"
-
 1. **Do not execute.**
 2. **Reclassify as an ambiguous mixed-intent request.**
 3. **Respond by describing intended interpretation and asking for approval.**
 
-**Example**  
-User: "Can you populate each locale file with missing data?"  
-AI:  
+**Example**
+User: "Can you populate each locale file with missing data?"
+AI:
 ```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)